packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -1,31 +1,28 @@
 .grav-c-list-cards-basic {
-  padding: 0;
-  list-style-type: none;
+  @include grav-l-column-list;
 
   > li + li {
     margin-top: $grav-sp-xl;
   }
 
   @media all and (min-width: grav-breakpoint(large)) {
-    display: flex;
-    flex-wrap: wrap;
+    @include grav-l-column-layout(3, $grav-sp-l);
+    @include grav-l-column-no-danglies;
+
     margin-right: -$grav-sp-l;
```

Comment:
If you tweak the `grav-l-column-layout()` mixin as I suggested above, you can remove most of these rules :-)